src/mongo/tree/MongoDatabaseTreeItem.ts

```diff
@@ -146,6 +147,21 @@ export class MongoDatabaseTreeItem implements IAzureParentTreeItem {
 	}
 }
 
+export function validateCollectionName(collectionName: string): string | undefined | null {
```

Comment:
Rename this to `validateMongoCollectionName` since it's exported. We don't want to accidently use this for DocDb or something weird like that